packages/apps/frontera/src/ui/presentation/Table/Table.tsx

```diff
@@ -489,10 +489,7 @@ const TableBody = <T extends object>({
                 ? (s) => {
                     row?.getToggleSelectedHandler()(s);
 
-                    /// @ts-expect-error improve this later
-                    const rowId = (row.original as unknown)?.id;
-
-                    onFullRowSelection?.(rowId);
+                    onFullRowSelection?.(row.original);
```

Comment:
why do we need to take the entire row props ?

Reply:
We pass the entire row props for flexibility and future-proofing. Different items may have unique identifiers (e.g., invoices use 'number' instead of 'id'). This approach allows us to:
- Access appropriate identifiers for each item type
- Adapt to future property needs without changing the function
- Use context-specific properties as needed
- Reduce coupling between the function and data structure
This generic approach makes the function more versatile and maintainable across various scenarios.